Changelog.md

```diff
@@ -4,6 +4,7 @@ Language Features:
 
 
 Compiler Features:
+ * StandardCompiler: do not run EVM bytecode code generation, if only Yul IR or EWasm output is requested.
```

Comment:
This change is not really user-facing, is it? If this is kept, please capitalize the `do`.